distribution/docker/build.gradle

```diff
@@ -144,25 +160,25 @@ task integTest(type: Test) {
 
 check.dependsOn integTest
 
-void addBuildDockerImage(final boolean oss) {
-  final Task buildDockerImageTask = task(taskName("build", oss, "DockerImage"), type: LoggedExec) {
-    dependsOn taskName("copy", oss, "DockerContext")
+void addBuildDockerImage(final boolean oss, final boolean ubi) {
+  final Task buildDockerImageTask = task(taskName("build", oss, ubi, "DockerImage"), type: LoggedExec) {
+    dependsOn taskName("copy", oss, ubi, "DockerContext")
     List<String> tags
     if (oss) {
       tags = [ 
-        "docker.elastic.co/elasticsearch/elasticsearch-oss:${VersionProperties.elasticsearch}",
-        "elasticsearch-oss:test" 
+        "docker.elastic.co/elasticsearch/elasticsearch-oss:${VersionProperties.elasticsearch}${ubi ? '-ubi7' : ''}",
```

Comment:
Do we anticipate publishing multiple _versions_ of the UBI image for any given version of Elasticsearch? I suspect if and when we bump from 7 to 8 we'd no longer publish 7 anymore.

Reply:
Yes, there's potential that we'd end up supporting both `ubi7` and `ubi8` images, even at the same time (given the length of life of `ubi7`).

Reply:
And I'd rather refactor it at that time when we need both `ubi7` and `ubi8` but for now we should tag the images appropriately.